sources/packages/forms/src/form-definitions/partnerinformationandincomeappeal.json

```diff
@@ -0,0 +1,664 @@
+{
+  "title": "Appeals - Partner Information and Income",
+  "name": "partnerInformationAndIncomeAppeal",
+  "path": "partnerinformationandincomeappeal",
+  "type": "form",
+  "display": "form",
+  "tags": [
+    "common"
+  ],
+  "components": [
+    {
+      "label": "HTML",
+      "labelWidth": "",
+      "labelMargin": "",
+      "tag": "h3",
+      "className": "category-header-medium primary-color",
+      "attrs": [
+        {
+          "attr": "",
+          "value": ""
+        }
+      ],
+      "content": "Family details",
+      "refreshOnChange": false,
+      "customClass": "",
+      "hidden": false,
+      "modalEdit": false,
+      "key": "html116",
+      "tags": [],
+      "properties": {},
+      "conditional": {
+        "show": null,
+        "when": null,
+        "eq": "",
+        "json": ""
+      },
+      "customConditional": "",
+      "logic": [],
+      "attributes": {},
+      "overlay": {
+        "style": "",
+        "page": "",
+        "left": "",
+        "top": "",
+        "width": "",
+        "height": ""
+      },
+      "type": "htmlelement",
+      "input": false,
+      "placeholder": "",
+      "prefix": "",
+      "suffix": "",
+      "multiple": false,
+      "defaultValue": null,
+      "protected": false,
+      "unique": false,
+      "persistent": false,
+      "clearOnHide": true,
+      "refreshOn": "",
+      "redrawOn": "",
+      "tableView": false,
+      "dataGridLabel": false,
+      "labelPosition": "top",
+      "description": "",
+      "errorLabel": "",
+      "tooltip": "",
+      "tabindex": "",
+      "disabled": false,
+      "autofocus": false,
+      "dbIndex": false,
+      "customDefaultValue": "",
+      "calculateValue": "",
+      "calculateServer": false,
+      "widget": null,
+      "validateOn": "change",
+      "validate": {
+        "required": false,
+        "custom": "",
+        "customPrivate": false,
+        "strictDateValidation": false,
+        "multiple": false,
+        "unique": false
+      },
+      "allowCalculateOverride": false,
+      "encrypted": false,
+      "showCharCount": false,
+      "showWordCount": false,
+      "allowMultipleMasks": false,
+      "addons": [],
+      "id": "eamewph",
+      "hideLabel": true
+    },
+    {
+      "label": "<strong>On my first day of class, I'll be:</strong>",
```

Comment:
Minor, removing the `<strong>` should not impact the UI. An existing CSS should be taken care of.